net-gateway/src/endpoints/filters/http_overview_filters/endpoint.rs

```diff
@@ -0,0 +1,42 @@
+use std::sync::Arc;
+
+use actix_web::get;
+use actix_web::web;
+use actix_web::HttpRequest;
+use actix_web::HttpResponse;
+use actix_web::Responder;
+use net_token_verifier::fusion_auth::fusion_auth_verifier::FusionAuthVerifier;
+
+use crate::authorization;
+
+use crate::config::Config;
+use crate::core::general_filters::GeneralFilters;
+use crate::core::service_request_management::service_request_manager::ServiceRequestManager;
+use crate::endpoints::filters::http_overview_filters::request::manager::HttpOverviewFilterManager;
+
+#[get("/filter/http_overview")]
+async fn get_http_overview_filters(
+    config: web::Data<Config>,
+    params: web::Query<GeneralFilters>,
```

Comment:
Are we really in need of this one?

Reply:
Why don't we? GeneralFilters is a pair of start_date and end_date for query the filters